app/src/main/java/com/fer1592/k8s_android_console/view/fragments/ClustersFragment.kt

```diff
@@ -77,6 +85,20 @@ class ClustersFragment : Fragment() {
             }
         }
 
+        // Sets observer to show progress bar when deleting a cluster
+        clustersViewModel.processingData.observe(viewLifecycleOwner) {
+            if (it) (activity as MainActivity).showLoading()
```

Comment:
maybe we could create a new fragment extension function so we just call showLoading(true/false) directly